book/_config.yml

```diff
@@ -0,0 +1,55 @@
+# Book settings
+# Learn more at https://jupyterbook.org/customize/config.html
+
+title: CVX Short Course
+author: CVX
+logo: static/logo.jpeg
+only_build_toc_files: true
+
+execute:
+  execute_notebooks: force   #auto   # only execute if there is at least on output cell missing
```

Comment:
I don't know how to read the double comment; does "force" or "auto" mean "only execute if there is at least on output cell missing"?

Reply:
The force mode just means that all notebooks are executed from scratch not using any cache from previous runs. The auto is a different mode but I don't use it. Given we the time to build the book, force is the safest option